pylint/checkers/utils.py

```diff
@@ -943,7 +943,12 @@ def is_from_fallback_block(node: nodes.NodeNG) -> bool:
         for import_node in other_body
     )
     ignores_import_error = _except_handlers_ignores_exception(handlers, ImportError)
-    return ignores_import_error or has_fallback_imports
+    ignores_module_not_found_error = _except_handlers_ignores_exception(
```

Comment:
https://github.com/PyCQA/pylint/blob/86c073e49fc099d8111da96cfbb2f30ccd659338/pylint/checkers/utils.py#L949-L953
Looking at `_except_handlers_ignores_exception` the `exception` parameter is actually made to be a `tuple`. I think it makes sense to merge this with `L945` and pass `(ImportError, ModuleNotFoundError)`.

While you're at it it, it would be appreciated if you could add the typing to the `exception` parameter. I think it should be:
`exception: Tuple[Type[ImportError], Type[ModuleNotFoundError]]`

Reply:
You're right, I've missed the `tuple` ! I took the liberty to rename the function as well as its second parameter according to its new behavior.

Reply:
Good idea!